cli/lib/db-utils.ts

```diff
@@ -0,0 +1,9 @@
+const { NODE_ENV } = process.env
+
+const CONNECTION_NAMES = {
+  development: 'Forum-User-Migration',
```

Comment:
Maybe add a docstring for why this is different in dev?

Reply:
Got me adding a whole section to the readme.